internal/scanner/scanner.go

```diff
@@ -120,12 +122,6 @@ func (s *Scanner) ScanNamespace(ctx context.Context, nsName string) error { //no
 			Int("policies skipped", policies.SkippedNum),
 		).Msg("policy count")
 
-	// create PolicyReport
-	namespacedsReport := report.NewPolicyReport(namespace)
-	namespacedsReport.Summary.Skip = policies.SkippedNum
-
-	// old policy report to be used as cache
-	previousNamespacedReport, err := s.policyReportStore.GetPolicyReport(nsName)
 	if errors.Is(err, constants.ErrResourceNotFound) {
```

Comment:
This `if` is a leftover from the cache feature. Right now `err` is already handled at line 114, it's about getting the policies for a Namespace, not about getting a PolicyReport